dask_sql/physical/rel/logical/aggregate.py

```diff
@@ -290,8 +305,8 @@ def _collect_aggregations(
             value = (input_col, output_col, aggregation_function)
             collected_aggregations[key].append(value)
             output_column_order.append(output_col)
-
-        return collected_aggregations, output_column_order
+        # mutated df - dono how to avoid this !!!
```

Comment:
You can remove this comment. It is totally fine to return it here and dask-sql does this all the time. Your solution is ok.

Reply:
👍 got it